components/dash-table/src/dash-table/handlers/cellEvents.ts

```diff
@@ -84,6 +84,7 @@ export const handleClick = (
             visibleColumns,
             viewport
         );
+        newProps.active_cell = clickedCell;
     } else {
         newProps.active_cell = clickedCell;
```

Comment:
Looks like this value is not assigned conditionally anymore so I would just pull it out before the if statement instead of duplicating it in the else block.

Reply:
right right, already pushed fix. thanks!